pkg/csi/cinder/openstack/openstack.go

```diff
@@ -149,13 +149,6 @@ func InitOpenStackProvider(cfgFiles []string, regionName string, httpEndpoint st
 			klog.Infof("metrics available in %q", httpEndpoint)
 		}()
 	}
-	if len(cfgFiles) == 0 {
-		klog.Infof("Using internal cloud config file: %s", cfgFiles)
-		if err := createCfgFile(configFiles[0], regionName); err != nil {
-			klog.Fatalf("Failed to create cloud config file: %v", err)
-		}
-		return
-	}
```

Comment:
With the removal of this, `cfgFiles` is not used in this function at all except for logging it? Should we remove the arg altogether and if we still need to log the value - do it at the call site of this function instead?

Reply:
Please retain this as is. It is code per upstream provider and is necessary from how we use it. 

It will also help manage conflicts in future pulls from upstream.